tests/rule_based_profiler/data_assistant/test_onboarding_data_assistant.py

```diff
@@ -317,6 +319,66 @@ def test_onboarding_data_assistant_result_batch_id_to_batch_identifier_display_n
     )
 
 
+@pytest.mark.integration
```

Comment:
```suggestion
@pytest.mark.big
```

We got rid of the `integration` marker as part of the testing overhaul effort.
If you update this branch with develop this integration marker will cause this test to break.